treeherder/log_parser/tasks.py

```diff
@@ -44,6 +47,9 @@ def parse_log(project, job_log_url, job_guid, check_errors=False):
             job_log_url['id']
         )
 
+        logger.debug("Downloading and extracting log information for %s "
```

Comment:
to be consistent with the other strings, I think this should be:
"... for guid %s "